app/forms/concept_form.rb

```diff
@@ -0,0 +1,13 @@
+class ConceptForm
```

Comment:
@lsitu I'm just wondering if it is necessary to add a separate app/models/forms/xxx_form.rb each time you add a new model. 
But it would be fine with me if other people is agree with that.